File: Java/ROUND_3_B/src/test/java/SorterTests.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertArrayEquals;

public class SorterTests {

	@Test
	public void bubblesortPutsArrayInAscendingOrder() {
		int[] array = {3,1,2};
		array = Sorter.sort(SortKind.BUBBLE, array);
		assertArrayEquals(new int[]{1,2,3}, array );
	}
	
	@Test
	public void quicksortPutsArrayInAscendingOrder() {
		int[] array = {3,1,2};
		array = Sorter.sort(SortKind.QUICK, array);
		assertArrayEquals(new int[]{1,2,3}, array );
	}
	
	@Test
	public void insertionsortPutsArrayInAscendingOrder() {
		int[] array = {3,1,2};
		array = Sorter.sort(SortKind.INSERTION, array);
		assertArrayEquals(new int[]{1,2,3}, array );
	}

}

File: Kotlin/ROUND_3_B/src/test/kotlin/SorterTests.kt
import org.junit.Test
import org.junit.Assert.assertArrayEquals


class SorterTests {
    @Test
    fun bubblesortPutsArrayInAscendingOrder() {
        var array = intArrayOf(3, 1, 2)
        array = sort(SortKind.BUBBLE, array)
        assertArrayEquals(intArrayOf(1, 2, 3), array)
    }

    @Test
    fun quicksortPutsArrayInAscendingOrder() {
        var array = intArrayOf(3, 1, 2)
        array = sort(SortKind.QUICK, array)
        assertArrayEquals(intArrayOf(1, 2, 3), array)
    }

    @Test
    fun insertionsortPutsArrayInAscendingOrder() {
        var array = intArrayOf(3, 1, 2)
        array = sort(SortKind.INSERTION, array)
        assertArrayEquals(intArrayOf(1, 2, 3), array)
    }
}